cookbook/doctrine/registration_form.rst

```diff
@@ -5,28 +5,36 @@
 How to Implement a simple Registration Form
 ===========================================
 
-Some forms have extra fields whose values don't need to be stored in the
-database. For example, you may want to create a registration form with some
-extra fields (like a "terms accepted" checkbox field) and embed the form
-that actually stores the account information.
+Creating a registration form is pretty easy - it *really* means just creating
+a form that will update some ``User`` object (a Doctrine entity in this example)
```

Comment:
I believe "model" is better to use as a generic name of objects that get persisted than the even more generic "object".